script/release.py

```diff
@@ -33,9 +31,7 @@
 
 
 def get_current_version():
-    current = io.open(os.path.join(os.path.dirname('__file__'), 'appium', 'version.py'), encoding='utf-8').read().rstrip()
-    print('The current version is {}, type a new one'.format(MESSAGE_YELLOW.format(current)))
-    return current
+    os.system('uv version')
```

Comment:
this method does not return anything, but looks like it should

Reply:
The original one was also to print the version in the stdout, so return value is not matter

Reply:
then the method name is confusing as `get` prefix implicitly assumes it must return something